.gitignore

```diff
@@ -82,6 +82,9 @@ target/
 profile_default/
 ipython_config.py
 
+# DJANGO Migrations
+migrations/
```

Comment:
You did this wrong I believe. We need `__init__.py` inside migrations. You should have seperate .gitignore inside the migrations folder configured to ignore everything except `__init__.py`.

Reply:
done